app/views/admin/enterprises/form/_shop_preferences.html.haml

```diff
@@ -1,104 +1,95 @@
-        'ta-toolbar' => "[['h1','h2','h3','h4','p'],['bold','italics','underline','clear'],['insertLink']]",
-        'placeholder' => t('.shopfront_message_placeholder')}
+  .three.columns.alpha
+    = f.label "enterprise_preferred_shopfront_message", t('.shopfront_message')
+  .eight.columns.omega
+    %text-angular{'ng-model' => 'Enterprise.preferred_shopfront_message', 'id' => 'enterprise_preferred_shopfront_message', 'name' => 'enterprise[preferred_shopfront_message]', 'class' => 'text-angular textangular-strip', 'ta-paste' => "stripFormatting($html)", "textangular-links-target-blank" => true,
+      'ta-toolbar' => "[['h1','h2','h3','h4','p'],['bold','italics','underline','clear'],['insertLink']]",
+      'placeholder' => t('.shopfront_message_placeholder')}
 .row
-  .alpha.eleven.columns
-    .three.columns.alpha
-      = f.label "enterprise_preferred_shopfront_closed_message", t('.shopfront_closed_message')
-    .eight.columns.omega
-      %text-angular{'ng-model' => 'Enterprise.preferred_shopfront_closed_message', 'id' => 'enterprise_preferred_shopfront_closed_message', 'name' => 'enterprise[preferred_shopfront_closed_message]', 'class' => 'text-angular textangular-strip', 'ta-paste' => "stripFormatting($html)", "textangular-links-target-blank" => true,
-        'ta-toolbar' => "[['h1','h2','h3','h4','p'],['bold','italics','underline','clear'],['insertLink']]",
-        'placeholder' => t('.shopfront_closed_message_placeholder')}
+  .three.columns.alpha
+    = f.label "enterprise_preferred_shopfront_closed_message", t('.shopfront_closed_message')
+  .eight.columns.omega
+    %text-angular{'ng-model' => 'Enterprise.preferred_shopfront_closed_message', 'id' => 'enterprise_preferred_shopfront_closed_message', 'name' => 'enterprise[preferred_shopfront_closed_message]', 'class' => 'text-angular textangular-strip', 'ta-paste' => "stripFormatting($html)", "textangular-links-target-blank" => true,
+      'ta-toolbar' => "[['h1','h2','h3','h4','p'],['bold','italics','underline','clear'],['insertLink']]",
+      'placeholder' => t('.shopfront_closed_message_placeholder')}
 .row
-  .alpha.eleven.columns
-    .three.columns.alpha
-      = f.label "enterprise_preferred_shopfront_taxon_order", t('.shopfront_category_ordering')
-      %br
-      (top to bottom)
-    .eight.columns.omega
-      %textarea.fullwidth{ id: 'enterprise_preferred_shopfront_taxon_order', name: 'enterprise[preferred_shopfront_taxon_order]', rows: 6, 'ng-model' => 'Enterprise.preferred_shopfront_taxon_order', 'ofn-taxon-autocomplete' => '', 'multiple-selection' => 'true', placeholder: 'Category' }
+  .three.columns.alpha
+    = f.label "enterprise_preferred_shopfront_taxon_order", t('.shopfront_category_ordering')
+    %br
+    (top to bottom)
```

Comment:
this is a missing translation.

Reply:
Done here: https://github.com/openfoodfoundation/openfoodnetwork/pull/6281

Reply:
awesome!